cdap-docs/user-guide/source/tutorials/logistic.rst

```diff
@@ -0,0 +1,237 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2017 Cask Data, Inc.
+    :description: The CDAP User Guide: Getting Started
+
+:hide-toc: true
+
+===============================================================
+Example: Building a Fraud Classification Machine Learning Model
+===============================================================
+
+Introduction
+------------
+In this tutorial, you will learn how to use CDAP to build and operationalize an enterprise-ready Machine Learning for prediciting credit fraud using only CDAP’s point-and-click interfaces.
+
+Scenario
+---------
+You are processing a large number of credit card transactions. Identfiying which transactions are fraudelent is business critical.
+
+- You want to train a Logistic Regression model using labeled data
+
+- You want to be able to send events to the model in realtime, and see which are predicted to be fraudelent
+
+Data
+----
+Click below to donwload a `.csv` file containing the data necessary to complete the tutorial.
```

Comment:
`Click the link below to download the fraud_train.csv` may be? Just to be consistent with the other tutorial